bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -1649,15 +1650,9 @@ void ReplicaImp::onInternalMsg(InternalMessage &&msg) {
 }
 
 std::string ReplicaImp::getReplicaLastStableSeqNum() const {
-  std::ostringstream oss;
-  std::unordered_map<std::string, std::string> result, nested_data;
-
-  nested_data.insert(toPair(getName(lastStableSeqNum), lastStableSeqNum));
-  result.insert(
-      toPair("sequenceNumbers ", concordUtils::kvContainerToJson(nested_data, [](const auto &arg) { return arg; })));
-
-  oss << concordUtils::kContainerToJson(result);
-  return oss.str();
+  nlohmann::json j;
+  j["sequenceNumbers"]["lastStableSeqNum"] = lastStableSeqNum;
```

Comment:
Just wondering, why can't you achieve the same without the JSON library?

Reply:
We can but why serialise JSON by hand when we have got a library for this in the project?